main/php.h

```diff
@@ -231,17 +231,7 @@ typedef unsigned int socklen_t;
 #include "zend_hash.h"
 #include "zend_alloc.h"
 #include "zend_stack.h"
-
-#if STDC_HEADERS
-# include <string.h>
-#else
-# ifndef HAVE_MEMCPY
```

Comment:
Remove HAVE_MEMCPY define in config.w32.h.in?

Reply:
Applied and removed in the next commit. The *nix build system will now still define the HAVE_MEMCPY symbol otherwise, but that is part of another macro - `AC_PROG_CC`, I think. So not related to this patch and doesn't cause issues.